src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -895,4 +895,11 @@ Do you want to proceed?</value>
   <data name="_0_bases" xml:space="preserve">
     <value>'{0}' bases</value>
   </data>
+  <data name="Rename_file" xml:space="preserve">
+    <value>_Rename file</value>
+  </data>
+  <data name="Rename_file_incompatible_type" xml:space="preserve">
+    <value>_Rename file (incompatible type)</value>
```

Comment:
could we just return the string we want to show if we want to disable?  for example "disabled due to multiple parts"?

Reply:
I'm trying to keep the strings shorter to avoid weird string wrapping in the UI, but also having string indication that the checkbox is disabled. 

"Rename file disabled due to multiple parts" I think is a bit long. I'm open to suggestions on this.